server/konnectors/malakoff_mederic.coffee

```diff
@@ -0,0 +1,174 @@
+                    "callCount=1\n\
+                    page=/espaceClient/LogonAccess.do\n\
+                    httpSessionId=#{httpSessionId}\n\
+                    scriptSessionId=#{scriptSessionId}\n\
+                    c0-scriptName=InternauteValidator\n\
+                    c0-methodName=checkConnexion\n\
+                    c0-id=0\n\
+                    c0-param0=boolean:false\n\
+                    c0-param1=string:#{requiredFields.login}\n\
+                    c0-param2=string:#{requiredFields.password}\n\
+                    batchId=0\n"
+
+            # Log in
+            request checkOption, (err, res, body) ->
+                if err
+                    log.error err
+                    return next 'request error'
+                else if res.statusCode >= 400
+                    log.error 'Authentication error'
+                    return next 'request error'
+                # The body should not contain LOGON_KO
+                else if body.indexOf('LOGON_KO') > -1
+                    log.error 'Authentication error'
+                    return next 'bad credentials'
+
+                log.info 'Logged in'
+
+                path = "/espaceClient/sante/tbs/redirectionAction.do"
+                reimbursementUrl = "#{domain}#{path}"
+                options =
+                    method: 'GET'
+                    url: reimbursementUrl
+                    jar: true
+
+                # Get the reimbursements
+                request options, (err, res, body) ->
+                    if err
+                        logger.error err
+                        return next 'request error'
+
+                    data.html = body
+                    next()
+
+
+# Parse the fetched page to extract bill data.
+parsePage = (requiredFields, healthBills, data, next) ->
+
+    healthBills.fetched = []
+    return next() if not data.html?
+
+    $ = cheerio.load data.html
+
+    $('.headerRemboursements').each ->
+        amount = $(this).find('.montant').text()
+        amount = amount.replace(' €', '')
+        amount = parseFloat amount
+
+        dateText = $(this).find('.dateEmission').text()
+        date = dateText.split('Emis le ')[1].split('aux')[0]
+
+        pdfUrl = $(this).find('#tbsRembExportPdf').attr('href')
+        pdfUrl = "#{domain}#{pdfUrl}"
+
+        bill =
+            amount: amount
+            type: 'health'
+            date: moment date, 'DD/MM/YYYY'
+            vendor: 'Malakoff Mederic'
+            pdfurl: pdfUrl
+
+        healthBills.fetched.push bill if bill.amount?
+    next()
+
+
+buildNotification = (requiredFields, healthBills, data, next) ->
+    log.info "Import finished"
+    notifContent = null
+    if healthBills?.filtered?.length > 0
+        localizationKey = 'notification malakoff'
+        options = smart_count: healthBills.filtered.length
+        healthBills.notifContent = localization.t localizationKey, options
+
+    next()
+
+
+fileOptions =
+    vendor: 'malakoff mederic'
```

Comment:
Sure. Shouldn't the string be without space as well? I guess this field is used for the filename ?